src/roman_datamodels/maker_utils.py

```diff
@@ -343,7 +343,7 @@ def mk_cal_step():
     -------
     roman_datamodels.stnode.CalStep
     """
-    calstep = stnode.CalStep()
+    calstep = stnode.CalStepValues()
```

Comment:
```suggestion
    calstep = stnode.CalStep()
```
VSCode got a bit too aggressive here with its variable rename tool I used in other parts of the code.